sdktests/common_tests_events_contexts.go

```diff
@@ -265,7 +265,11 @@ func (c CommonEventTests) EventContexts(t *ldtest.T) {
 							_ = basicEvaluateFlagWithOldUser(t, client, debuggedFlagKey, user, defaultValue)
 							client.FlushEvents(t)
 							payload := events.ExpectAnalyticsEvents(t, defaultEventTimeout)
-							m.In(t).Assert(payload, m.ItemsInAnyOrder(debugEventMatcher, IsSummaryEvent()))
```

Comment:
This expectation was wrong for client-side SDKs because we have explicitly sent an identify event in this payload.